app/javascript/components/users-dropdown.vue

```diff
@@ -0,0 +1,49 @@
+<template>
+  <div>
+    <clickable-dropdown
+      :body-title="dropdownTitle"
+      :no-items-message="noUsersMessage"
+      :items="users"
+      @item-clicked="onItemClicked"
+    /><br>
+    <div v-for="(user, index) in selected"
```

Comment:
**[eslint]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub>
'v-for' should be on a new line                                                   vue/max-attributes-per-line